src/Settings/Default.hs

```diff
@@ -96,9 +89,10 @@ defaultRtsWays = do
     mconcat
         [ pure [ logging, debug, threaded, threadedDebug, threadedLogging ]
         , (profiling `elem` ways) ? pure [threadedProfiling]
-        , (dynamic `elem` ways) ?
-          pure [ dynamic, debugDynamic, threadedDynamic, threadedDebugDynamic
-                 , loggingDynamic, threadedLoggingDynamic ] ]
+        -- , (dynamic `elem` ways) ?
```

Comment:
why is `dynamic` commented out?

Reply:
leftover from whene things would not build, fixed now thanks!